src/stores/ContractsStore.ts

```diff
@@ -59,10 +59,17 @@ class ContractsStore extends PersistableStore {
         WalletStore.account
       )
 
-    await this.connectedAccounts[WalletStore.account].getOwnedERC721(
-      this.currentBlock,
-      this.network
-    )
+    if (!this.addressToTokenIds || accountChange) {
+      this.addressToTokenIds = this.connectedAccounts[
+        WalletStore.account
+      ].syncAddressToTokenIds(this.currentBlock, this.network)
+    } else {
```

Comment:
Let's not do `else`, let's do return in the upper `if`

Reply:
Resolved